### DISCUSSION TITLE
This issue is back with the Vimeo player:

### DISCUSSION THREAD

**Discussion by Davi-T** [vimeo/player.js#1015] [Bug reports]
This issue is back with the Vimeo player:

<img width="1285" alt="Screen Shot 2022-07-20 at 8 59 31 AM" src="https://user-images.githubusercontent.com/215157/180028767-31848699-ecef-4b34-8e23-cf4136ba3023.png">

_Originally posted by @brown2020 in https://github.com/vimeo/player.js/issues/553#issuecomment-1190464477_

Can we get a fix pretty please?

**Comment by Davi-T** [comment]
The error I see on my end. Appears in vendor.module.js >
<img width="1408" alt="js_console_error" src="https://user-images.githubusercontent.com/20928342/219966842-4867aeca-27f3-4815-ae1c-d1c43f582014.png">

**Comment by Davi-T** [comment]
Issue can be seen by going to my site, and clicking any one of the project thumbnails below (on Desktop. Mobile seems to work fine). Once clicked, a modal should appear and a Vimeo video should be playing back in its own player.  This has been working fine and dandy for years, until now :(

www.Davi-T.com

**Comment by rkrishnan8594** [comment]
I'm not observing this issue on your website. Can you provide an actively reproducible example?

**Comment by Davi-T** [comment]
That's because I adjusted it to bypass that issue (now requires you to click the video to start, but at least the video appears now). That issue still very much exists though. From what I can remember, it had to do with autoplay muted attributes. Previously, it would work fine but then it wouldn't even load the video after the last update.

**Comment by dannyfiresnake** [comment]
I see this on https://roon.app/ -- no bypass, so you can see it.

In the JS Console in the latest Chrome on MacOS, I see 12 warnings, all from vendor.module.js (from Vimeo).

my embed of vimeo looks like this (Vue code):


```html
<iframe :src="s.video.embedUrl + '&byline=false&loop=true&pip=false&muted=true&autoplay=true'"
        frameborder="0" allow="autoplay; fullscreen; picture-in-picture"
        style="position:absolute;top:0;left:0;width:100%;height:100%;"></iframe>
```

The embedUrl is from the Vimeo API. For example: https://player.vimeo.com/video/835585753?h=f95b1b51b0
